### ISSUE TITLE
teach packrat about RSPM linux binaries

### ISSUE THREAD

**Issue by slopp** [rstudio/packrat#570]
RSPM repositories can provide users linux package binaries if users set the repo option correctly. 

For examples, see: http://cluster.rstudiopm.com/client/#/repos/1/overview

Packrat should remove the __linux__ component of the URL to ensure the projects remain portable across distros.

**Comment by colearendt** [comment]
To be clear, I believe you are talking about removing the linux component of the URL on _snapshot_ only.

Wouldn't this require removing `/__linux__/bionic`, for example?

**Comment by slopp** [comment]
Yes, sorry linux component means `__linux__/<thing that follows>`

The transaction ID or `/latest` that follows should be retained.
